quickstart/101-virtual-network-manager-create-mesh/readme.md

```diff
@@ -0,0 +1,28 @@
+# Azure resource group
+
+This template deploys an Azure Virtual Network Manager instance with a connectivity configuration for a Mesh topology. It includes resources including virtual networks, subnets, and more.
+
+## Terraform resource types
+
+- [random_pet](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet)
+- [azurerm_resource_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group)
+- [azurerm_virtual_network](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_network)
+- [azurerm_subnet](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet)
+- [random_id](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/id)
```

Comment:
Unless I am missing it, this is not used in this config. Please remove if not used.